datasets/air_dialogue/README.md

```diff
@@ -0,0 +1,194 @@
+```
+
+### Data Fields
+
+BuilderConfig: `air_dialogue_data`:
+
+key name | Description |
+|---|---|
+|'action' |
+|'intent' |
+|'timestamps' | 
+|'dialogue' | 
+|'expected_action' |
+|'correct_sample' |
```

Comment:
could you add brief descriptions here please ?

Reply:
done